tests/scripts/all.sh

```diff
@@ -2206,6 +2208,9 @@ config_psa_crypto_config_all_ec_algs_use_psa () {
         scripts/config.py unset MBEDTLS_ECDH_C
         scripts/config.py unset MBEDTLS_ECJPAKE_C
         scripts/config.py unset MBEDTLS_ECP_C
+        scripts/config.py unset MBEDTLS_ECP_LIGHT
```

Comment:
Wait, this will be the next goal, not the current one. [For now](https://github.com/orgs/Mbed-TLS/projects/1#column-19391690) we just want things to work with `ECP_LIGHT && !ECP_C`. Getting things to work without `ECP_LIGHT` will be [the next EPIC](https://github.com/orgs/Mbed-TLS/projects/1#column-19391691).